test/config_test.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+/**
+ * Load all configurations from a tar file and make sure they are valid.
+ * @return uint32_t the number of configs tested.
+ */
+uint32_t runConfigTest();
```

Comment:
Suggest wrapping in a namespace to avoid pollution. Now that we're supporting a single test binary again, there's more room for colliding declarations unless they are namespace qualified.